cnd/src/ethereum.rs

```diff
@@ -60,10 +178,10 @@ pub struct Log {
 #[derive(Debug, Default, Clone, PartialEq, Deserialize)]
 pub struct Block {
     /// Hash of the block
-    pub hash: Option<H256>,
+    pub hash: Option<Hash>,
```

Comment:
Can you record a follow-up issue for making this not an `Option`? It is so annoying :grimacing:

Reply:
recorded: https://github.com/comit-network/comit-rs/issues/2414